tests/test_parser.py

```diff
@@ -1403,6 +1403,57 @@ def test_mls_7():
     assert str(mls.value) == '"This" is a "mls" on one "line"'
 
 
+def test_mls_8():
+    statements = parse_code(
+        """
+\"""String: ""\"""
+"""
+    )
+    assert len(statements) == 1
+    mls = statements[0]
+
+    assert isinstance(mls, LocatableString)
+    assert mls.lnr == 2
+    assert mls.elnr == 2
+    assert mls.start == 1
+    assert mls.end == 17
+    assert str(mls.value) == 'String: ""'
```

Comment:
It's interesting to see how GitHub seems to have issues with identifying mls strings for syntax highlighting as well. Guess we shouldn't feel to bad then.